easybuild/easyconfigs/h/HDF5/HDF5-1.8.17-GCC-5.4.0-2.26-serial.eb

```diff
@@ -0,0 +1,26 @@
+name = 'HDF5'
+version = '1.8.17'
+versionsuffix = '-serial'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
`optarch` is enabled by default, you should set `usempi` to `False` for a serial build?

Reply:
I am building with `GCC` toolchain and `usempi` is defined only for toolchains with `MPI`.
I will go ahead and clean the `optarch`.

Reply:
Good point, sorry about missing that.